db/migrate/20180126101234_add_unconfirmed_email.rb

```diff
@@ -0,0 +1,16 @@
+# as seen here: https://github.com/plataformatec/devise/wiki/How-To:-Add-:confirmable-to-Users
+class AddUnconfirmedEmail < ActiveRecord::Migration[5.1]
+  def up
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_column :users, :unconfirmed_email, :string
+    add_index :users, :confirmation_token, unique: true
+    User.all.update_all confirmed_at: DateTime.now
+  end
+
+  def down
```

Comment:
I think you can make this migration shorter by using `reversible` and only specifying up with update all. That way you can use the `change` method.

Reply:
Yes, I could, but I see no reason. Copying it from the webpage referenced worked aswell. But will not prevent you from tidying this up :D